### PR TITLE
Migrate gringotts to lodash@4.17.4

### PR DESCRIPTION
**Pull request by bzalasky**

### CODE REVIEW

File: lib/mixin-helper.coffee [hunk 1]
```diff
@@ -1,5 +1,17 @@
 Chaplin = require 'chaplin'
 
+oldLodashFunctions = (object) ->
```
Comment (eugenet8k) [inline lib/mixin-helper.coffee]: maybe just call it `functions` with a comment on top that it's taken from LoDash 3.x

File: package.json [hunk 2]
```diff
@@ -41,12 +41,12 @@
   "dependencies": {
     "backbone": "1.3.3",
     "backbone-validation": "0.11.5",
-    "backbone.stickit": "0.9.2",
+    "backbone.stickit": "https://github.com/NYTimes/backbone.stickit.git",
```
Comment (eugenet8k) [inline package.json]: Is this because they didn't release something?

File: package.json [hunk 3]
```diff
@@ -41,12 +41,12 @@
   "dependencies": {
     "backbone": "1.3.3",
```
Comment (eugenet8k) [inline package.json]: Do we need to update backbone? I am not sure this one supports lodash 4.x. The same with Chaplin

File: package.json (hunk 2 above)
Comment (bzalasky) [reply package.json]: Master has the commits for supporting lodash@4.x, and hasn't been released.

File: package.json (hunk 3 above)
Comment (bzalasky) [reply package.json]: Lodash v4 works with Backbone ≥ v1.3.0.

File: lib/mixin-helper.coffee (hunk 1 above)
Comment (bzalasky) [reply lib/mixin-helper.coffee]: Will do.

**Comment by bzalasky** [comment]
Everything should be addressed now.

**Pull request merged**